### PR TITLE
Starting the Bio for githublauren-bio.md

### PR DESCRIPTION
**Pull request by laurenslocum**
## Start of a bio

Check list for things that need to happen before code ships.
- [ ] get feedback from at least one team member
- [ ] add favorite restaurant
- [ ] add more hobbies

### CODE REVIEW

File: githublauren-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# About Lauren
+
+- Lives in Arlington, VA
```
Comment (ffleige) [inline githublauren-bio.md:5]: Greetings from Germany ;-)

Comment (laurenslocum) [inline githublauren-bio.md:5]: Hello ffleige in Germany!

**Comment by laurenslocum** [comment]
Before merging we need a thumbs up.

**Comment by laurenslocum** [comment]
:8ball:

**Pull request merged**